source/draw/DrawContext.ooc

```diff
@@ -38,12 +38,6 @@ DrawContext: abstract class {
 	createYuv420Semiplanar: abstract func (size: IntVector2D) -> Image
 	createYuv420Semiplanar: abstract func ~fromImages (y, uv: Image) -> Image
 	createYuv420Semiplanar: abstract func ~fromRaster (raster: RasterYuv420Semiplanar) -> Image
-	createImageFromStringAndFont: virtual func (message: String, fontAtlas: Image) -> Image {
-		target := this createMonochrome(This getStringPixelBounds(message, fontAtlas))
-		target fill(ColorRgba black)
-		target write(message, fontAtlas, IntPoint2D new(0, 0))
-		target
-	}
```

Comment:
`this createMonochrome` will always create a GPU image from a GPU context. Now, we always create a CPU image no matter what context so this level of abstraction was too high.